project/Dependencies.scala

```diff
@@ -137,6 +141,8 @@ object Dependencies {
     akkaTestkit,
     sprayTestkit,
     mockserverNetty,
-    mockito
+    mockito,
+    workbenchUtil,
```

Comment:
I don't buy that that definition of `rawlsCoreDependencies` is right rather than just lazy.